src/neptune/new/types/type_casting.py

```diff
@@ -111,12 +121,17 @@ def cast_value_for_extend(values: Union[Namespace, Series, Collection[Any]]) ->
     elif is_float_like(sample_val):
         return FloatSeries(values=values)
     elif is_string_like(sample_val):
-        warn_once(
-            message="The object you're logging will be implicitly cast to a string."
-            " We'll end support of this behavior in `neptune-client==1.0.0`."
-            " To log the object as a string, use `str(object)` instead.",
-            stack_level=4,
-        )
+        import sys
+        import traceback
+
+        traceback.print_stack(file=sys.stdout)
```

Comment:
Why do we print stack for extend, but not for regular cast?

Reply:
Ouch! That's debug. Good catch

Reply:
Fixed